Pocket/pocket-ios
# fix(braze): ensure that braze data is wiped and disabled on logout

## Summary

We need to ensure that braze is disabled and user data is wiped when there is no user session active.

## References 
This calls https://braze-inc.github.io/braze-swift-sdk/documentation/brazekit/braze/wipedata() when the session is null.

## Implementation Details
* Add a call to wipe data in braze

Note: This will mean that we can not do push notifications or In App messages unless a user is logged in.

## Test Steps
* Log out of Pocket
* Ensure that you can not send a push notification to the logged out device

## PR Checklist:

## Screenshots
